README.md

```diff
@@ -6,6 +6,8 @@
 
 Document management service
```

Comment:
I'd add a sentence or two to be more specific - "document management service" is rather generic. Samba for instance is a document management service :)